### ISSUE TITLE
Add support for Cobol

### ISSUE THREAD

**Issue by catno75** [judge0/api#159]
This feature will be sponsored by Evalart.

**Comment by hermanzdosilovic** [comment]
Hi @catno75,

Thank you for considering sponsoring this feature. I can release this by the end of April 2020 for a $40 donation.

**Comment by hermanzdosilovic** [comment]
Hi @catno75,

This feature is currently in progress and I would just like to give an update.

I have successfully compiled and installed [GnuCobol 2.2](https://ftp.gnu.org/gnu/gnucobol). I am aware that there is a version [3.0-rc1](https://sourceforge.net/projects/open-cobol/files/gnucobol/3.0), but since it is still a release candidate I do not want to add.

Submitted source code is saved as `main.cob` and here is a simple hello, world program:
```
IDENTIFICATION DIVISION.
PROGRAM-ID. MAIN.
PROCEDURE DIVISION.
DISPLAY "hello, world".
STOP RUN.
```

I am then compiling it with the:
```
cobc -free -x $ARGS main.cob
```
where `$ARGS` are custom optional command line arguments that can be used.

After that execution is as follows:
```
./main
```

Can you please just confirm that this an expected behavior? :)

**Comment by catno75** [comment]
Hi Herman,

looks good. The API would not be aware of the compiling and execution
stepts right? I mean, the user would send the code and the API would return
the output, like hello world in your example?

Regards,

Gino

**Comment by hermanzdosilovic** [comment]
Yes, this works as for any other available language that is already supported.

**Comment by catno75** [comment]
Perfect, we are good then!

**Comment by hermanzdosilovic** [comment]
Hi @catno75 ,

thanks to [Evalart](https://evalart.com) for sponsoring this feature for 40 USD.

This language is now available in [v1.7.0](https://github.com/judge0/api/releases/tag/v1.7.0) and you can try it [here](https://ide.judge0.com/?Pzdg).

**Comment by catno75** [comment]
Great, thanks Herman!
